spinnaker_components/ensemble/recording.c

```diff
@@ -1,20 +1,20 @@
 #include "recording.h"
 
+// Generic buffer initialisation
 bool record_buffer_initialise(recording_buffer_t *buffer, address_t region,
-                              uint n_blocks, uint n_neurons) {
-  // Generate and store buffer parameters
-  buffer->block_length = (n_neurons >> 5) + (n_neurons & 0x1f ? 1 : 0);
+                              uint32_t n_blocks, uint32_t block_length)
+{
+  // Store buffer parameters
   buffer->n_blocks = n_blocks;
-  buffer->_sdram_start = (uint *) region;
+  buffer->block_length = block_length;
```

Comment:
I'd have assumed this was in bytes if I hadn't read further down - rename to block_length_words, block_words or something?